ambry-api/src/main/java/com.github.ambry/clustermap/ClusterMap.java

```diff
@@ -26,23 +26,23 @@
 
   /**
    * Gets PartitionId based on serialized bytes.
-   * @param stream data input stream that contains the serialized partition bytes
+   * @param stream {@link InputStream} that contains the serialized partition bytes
    * @return deserialized PartitionId
    */
-  public PartitionId getPartitionIdFromStream(DataInputStream stream) throws IOException;
+  PartitionId getPartitionIdFromStream(InputStream stream) throws IOException;
```

Comment:
Because this is implementation dependent. Internally they may (and do) use a common method in `ClusterMapUtils`.